File: manage-payments/python/retrieve.py
# The sandbox API key passed in 'authorization' is public.
# Don't submit any personally identifiable information in any requests made with this key.
# Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
# for secure testing.

import http.client
import json

conn = http.client.HTTPSConnection("api.dojo.tech")

headers = {
    'Version': "2022-04-07",
    'Authorization': "Basic <your_api_key>"
    }

conn.request("GET", f"/payment-intents/{paymentIntentId}", headers=headers)

res = conn.getresponse()
data = res.read()
resp_data = {}
resp_data['status'] = json.loads(data)["status"]

print(resp_data)

conn.close()

File: manage-payments/php/retrieve.php
<?php
// The sandbox API key used in this example is public.
// Don't submit any personally identifiable information in any requests made with this key.
// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
// for secure testing.

namespace Test;
require_once "vendor/autoload.php";

use Dojo_PHP\ApiFactory;

$apiKey = "<your_api_key>";
$api = ApiFactory::createPaymentIntentApi($apiKey);

$pi = $api->paymentIntentsGet("<paymentIntentId>", \Dojo_PHP\API_VERSION);